### PR TITLE
Created testing environment and index page setup

### PR DESCRIPTION
**Pull request by jocontreras**
As a visitor. So that I know I´m on the index page. I would like to see the text Craft Academy Mailboxer.  
Added cucumber, rspec and coveralls.
Added Rspec, Cucumber files.  
Added Coveralls gem.

Ready for review! 
@tochman @diraulo @amber

### CODE REVIEW

**Comment by casperjureskog** [comment]
i think that travis is failing due to not yet being implemented i the gems

File: .rspec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```
Comment (AmberWilkie) [inline .rspec:3]: This file should have a `--format documentation` line in it.

File: config/secrets.yml [hunk 2]
```diff
@@ -5,16 +5,16 @@
...
 development:
-  secret_key_base: b7c5064e58eeaa53fe778a48aee0646375db6b41a9bacc16023fc54b84daf5aa2caeb03055366b2c8bc2695463ccb585b7fecb3fe937ba119027226b68bf56e0
+  secret_key_base: cd43f7ffa50962b64abb4dfbf53eb061218c9892d2626b0f4bd40b5119c3fd8407b74406183b6bd916915903dd9e6257db50d5dca6e165873a8e9133b01efd27
 
 test:
-  secret_key_base: dcf2c74608b0f94135c3e5be443535ad426e82027b88d399089df79f518a129ab7cde073ef96119ee555544ecbf9ecc6433fc686e793394d141c0b3dc8d80aa8
+  secret_key_base: dd7ca166b5757049e80c6bce45b20f03c398680e9e5a9080e2d925a20da873a2d273484f9f5b2d8345e78e22bfe1316f363f1c6748b759a0158251a58c557385
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
```
Comment (AmberWilkie) [inline config/secrets.yml]: This file should be excluded from version control: add it to your `.gitignore` file and make sure the PR is updated (that is, when you come back to this page you shouldn't see it in the list of files).

File: features/support/env.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+#   Before('~@no-txn', '~@selenium', '~@culerity', '~@celerity', '~@javascript') do
+#     DatabaseCleaner.strategy = :transaction
+#   end
+#
+
+# Possible values are :truncation and :transaction
+# The :transaction strategy is faster, but might give you threading problems.
+# See https://github.com/cucumber/cucumber-rails/blob/master/features/choose_javascript_database_strategy.feature
+Cucumber::Rails::Database.javascript_strategy = :truncation
+
```
Comment (AmberWilkie) [inline features/support/env.rb]: Delete the comments out of this file.

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  #
+  # The different available types are documented in the features, such as in
+  # https://relishapp.com/rspec/rspec-rails/docs
+  config.infer_spec_type_from_file_location!
+
+  # Filter lines from Rails gems in backtraces.
+  config.filter_rails_from_backtrace!
+  # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace("gem name")
+end
```
Comment (AmberWilkie) [inline spec/rails_helper.rb]: Same thing here, clean up.

File: spec/spec_helper.rb [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
+end
```
Comment (AmberWilkie) [inline spec/spec_helper.rb]: And here.

**Review by AmberWilkie — COMMENTED** [review]
I honestly don't know anything about this switching from Rails 4 to 5, but there are some other things I want you guys to look at.

**Comment by tochman** [comment]
Now, Ambers comments needs to be addressed promptly. Also, when we, the Coaches, are asked to do a review and make comments on a PR, especially after office hours, we would like you guys to acknowledge that you have seen the comments by at least responding to them.

**Pull request merged**